<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassAdd</class>
 <widget class="QDialog" name="ClassAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="ok">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOper">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>请输入要添加的班级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>班级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>班级</string>
   </property>
  </widget>
  <widget class="QToolButton" name="out">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addClass">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCheck">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>请点击查看已有班级</string>
   </property>
  </widget>
  <widget class="QComboBox" name="classId">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>out</sender>
   <signal>clicked()</signal>
   <receiver>ClassAdd</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
